src/usdb_syncer/sync_meta.py

```diff
@@ -33,40 +33,56 @@ def __str__(self) -> str:
 class ResourceFile:
     """Meta data about a local file."""
 
-    fname: str
-    mtime: int
-    resource: str
+    fname: str | None
+    mtime: int | None
+    resource: str | None
```

Comment:
This is unfortunately allows many invalid configurations. Better to have a wrapper witht two fields, JobResult and the old ResourceFile.